volatility3/framework/plugins/mac/kauth_scopes.py

```diff
@@ -80,7 +80,7 @@ def _generator(self):
                 (
                     identifier,
                     format_hints.Hex(scope.ks_idata),
-                    len([l for l in scope.get_listeners()]),
+                    len([li for li in scope.get_listeners()]),
```

Comment:
Huh, good catch, I generally keep an eye out for single letter variables myself!  Might prefer listener, but you've fixed up so much I'm not gonna be that picky.  5;)

Reply:
its `listener` now :)